test/suite/test_backup13.py

```diff
@@ -0,0 +1,166 @@
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger, wttest
+import os, shutil
+from helper import compare_files
+from suite_subprocess import suite_subprocess
+from wtdataset import simple_key
+from wtscenario import make_scenarios
+
+# test_backup13.py
+# Test cursor backup with a block-based incremental cursor and force_stop.
+class test_backup13(wttest.WiredTigerTestCase, suite_subprocess):
+    conn_config='cache_size=1G,log=(enabled,file_max=100K)'
+    dir='backup.dir'                    # Backup directory name
+    logmax="100K"
+    uri="table:test"
+    nops=1000
+    mult=0
+
+    pfx = 'test_backup'
+    # Set the key and value big enough that we modify a few blocks.
+    bigkey = 'Key' * 100
+    bigval = 'Value' * 100
+
+    def add_data(self, uri):
+
+        c = self.session.open_cursor(uri)
+        for i in range(0, self.nops):
+            num = i + (self.mult * self.nops)
+            key = self.bigkey + str(num)
+            val = self.bigval + str(num)
+            c[key] = val
+        self.session.checkpoint()
+        c.close()
+        # Increase the multiplier so that later calls insert unique items.
+        self.mult += 1
+
+    def test_backup13(self):
+
+        self.session.create(self.uri, "key_format=S,value_format=S")
+        self.add_data(self.uri)
+
+        # Open up the backup cursor. This causes a new log file to be created.
+        # That log file is not part of the list returned. This is a full backup
+        # primary cursor with incremental configured.
+        os.mkdir(self.dir)
+        config = 'incremental=(enabled,granularity=1M,this_id="ID1")'
+        bkup_c = self.session.open_cursor('backup:', None, config)
+
+        # Add more data while the backup cursor is open.
+        self.add_data(self.uri)
+
+        # Now copy the files returned by the backup cursor.
+        all_files = []
+        while True:
+            ret = bkup_c.next()
+            if ret != 0:
+                break
+            newfile = bkup_c.get_key()
```

Comment:
That did not work, perhaps because backup cursors have no values and get an error back when one tries to do a `get_value`. So I added a comment about it.